securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -60,6 +60,35 @@ can pass `login-required` or `check-sso` to the init function. `login-required`
 or display the login page if not. `check-sso` will only authenticate the client if the user is already logged-in, if the user is not logged-in the browser will be
 redirected back to the application and remain unauthenticated.
 
+You can configure a _silent_ `check-sso` option.
+With this feature enabled, your browser won't do a full redirect to the {project_name} server and back to your application, but this action will be performed in a hidden iframe, so your application resources only need to be loaded and parsed once by the browser when the app is initialized and not again after the redirect back from {project_name} to your app.
+This is particular useful in case of SPAs (Single Page Applications).
```

Comment:
Sorry, maybe I missed the change, but it still shows up here.  Should be "This is particularly...

Reply:
As this PR was already closed and merged, I fixed it in a new PR #725 and also @abstractj made a PR #726 
These PRs are not yet merged, so you can't see the fix in other resources than there.

@andymunro You are mentioned as a reviewer in #726, so please confirm the fix there. I'll close #725 as a duplicate.